docs/quickstart.md

```diff
@@ -0,0 +1,108 @@
+# Goose in 5 minutes
+
+## Quickstart guide
+
+Goose is a developer agent that supercharges your software development by automating an array of coding tasks directly within your terminal or IDE. You can get it setup and running on your devices in only a few minutes.
+
+### Installation
+
+To install Goose, use `pipx`. First ensure [pipx][pipx] is installed:
+
+``` sh
+brew install pipx
+pipx ensurepath
+```
+
+Then install Goose:
+
+```sh
+pipx install goose-ai
+```
+
+### Running Goose
+
+#### Start a session
+
+From your terminal, navigate to the directory you'd like to start from and run:
+
+```sh
+goose session start
+```
+
+#### Set up a provider
+Goose works on top of LLMs and by default, it uses `openai` as the LLM provider but you can customize it as needed. You'll be prompted to set an [OPENAI_API_KEY][openai-key] if you haven't set one previously.
+
+>[!TIP]
+> **Billing:**
+>
+> You will need to add credits to your Open AI accounts to be able to successfully make requests.
+>
+
+
+#### Make Goose do the work for you
+You will see the Goose prompt `G❯`:
+
+```
+G❯ type your instructions here exactly as you would tell a developer.
+```
+
+Now you are interacting with Goose in conversational sessions - think of it like you're giving directions to a junior developer. The default toolkit allows Goose to take actions through shell commands and file edits. You can interrupt Goose with `CTRL+D` or `ESC+Enter` at any time to help redirect its efforts.
+
+#### Exit the session
+
+If you are looking to exit, use `CTRL+D`, although Goose should help you figure that out if you forget.
```

Comment:
```suggestion
To exit, use `CTRL+D`, although Goose should help you figure that out if you forget.
```

Reply:
how would it help them figure it out? would be nice to provide that info if we're gonna state this

Reply:
maybe remove the "although..." if we aren't gonna tell them how

Reply:
```suggestion
If you are looking to exit, use `CTRL+D`.
```